lib/geocoder/lookups/olleh.rb

```diff
@@ -135,20 +138,23 @@ def results(query)
       end
 
       case Olleh.check_query_type(query)
-      when "geocoding"
+      when 'addr_local_search'
+        return [] if doc["RESULTDATA"]["place"]["TotalCount"].to_i == 0
+        return doc["RESULTDATA"]["place"]["Data"]
+      when 'geocoding'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.